pymeasure/instruments/hp/hp3478A.py

```diff
@@ -0,0 +1,608 @@
+#
```

Comment:
This file is not part of your this pull request and it is actually part of the master branch.
Again, something wrong with rebase ?